autoscheduler/frontend/src/components/GenericCard/GenericCard.css

```diff
@@ -2,7 +2,7 @@
     display: inline-flex;
     flex-direction: column;
     align-items: center;
-    font-family: 'Roboto';
+    font-family: "Roboto", Arial, Helvetica, sans-serif;
```

Comment:
Completely unrelated, but I thought I'd fix it while I was working. Basically just adding a fallback font in case Roboto is ever unavailable for some reason. I could leave it out of this PR if you'd like, I just didn't think it was worth a separate one